### PR TITLE
Added number of licenses used field to O365 Org model & licensed field to O365 User model.

### PR DESCRIPTION
**Pull request by Juchazari**
http://doc.10.api.iland.test/1.0/#/Office%20365%20Backup/getOrganization

### CODE REVIEW

**Review by iamtash — APPROVED** [review]

File: src/sdk/model/o365/o365-user.ts [hunk 1]
```diff
@@ -57,6 +57,14 @@ export class O365User {
     return this._json.archived;
   }
 
+  /**
+   * Get whether this O365 user is an active user. False for active.
```
Comment (iamtash) [inline src/sdk/model/o365/o365-user.ts:61]: wrong doc. should be "Get whether this O365 user is a licensed user in the Microsoft 365 organization it belongs. False for unlicensed."

**Review by iamtash — COMMENTED** [review]
update commit msg and PR title to flip naming to "O365 Org" & "O365 User" to keep with the naming convention

**Comment by iamtash** [comment]
perfect thanks Julio. i know you added on `O65User.licensed` later but going forward since when you're updating more than one model/resource it's helpful to include swagger references for all, so you'd just add a second link to this http://doc.10.api.iland.test/1.0/#/Office%20365%20Backup/getUsers in your description

**Review by iamtash — APPROVED** [review]

**Comment by Juchazari** [comment]
Ah yes, thank you for the reminder @iamtash!

**Review by ariquinones — APPROVED** [review]

**Pull request merged**